src/DssSpell.sol

```diff
@@ -19,6 +19,11 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
+// Note: ignored on Goerli
+// interface ProxyLike {
+//     function exec(address target, bytes calldata args) external payable returns (bytes memory out);
+// }
```

Comment:
**nitpick**: Dead code
This code is not used, in previous spells sometimes it's included, sometimes not. I would prefer to remove it since: 
  - validating and checking this has no point
  - info contained in the comment (code) is not relevant to the task at hand (which is perform actions on goerli that can be performed)